src/graphql.rs

```diff
@@ -120,6 +123,10 @@ pub(super) struct Query(
     node::NodeStatusQuery,
     qualifier::QualifierQuery,
     outlier::OutlierQuery,
+);
+
+#[derive(MergedObject, Default)]
+struct SubQueryTwo(
```

Comment:
You mentioned you split the query into these two parts to avoid the error. Could you tell me the criteria you used to do so? Is there any other method besides this split?

Reply:
Currently, I split the structure in half, the smallest number of splits that won't cause an error, without any special criteria. I thought about splitting the structures by function, but I thought that would lead to a lot of meaningless structures whose sole purpose is to implement `MergedObject` trait,

Reply:
This looks very weird to me. It's not ideal to use arbitrary code to avoid errors. Splitting the query into two subqueries without any reasonable basis doesn't make sense.

Reply:
I spoke with @sehkone about the splitting structs and here's what we talked about.
1. Use the split structures defined by the current pr (`SubMutationOne`, `SubMutationTwo`, `SubQueryTwo`, `SubQueyrTwo`).
2. Add more details in the changelog about what led to splitting the structure.
  - Add description of `MergedObject` trait causing "overflow evaluating the requirement" error.
  - Add clarification that they should be merged back into a single structure after fixing a bug in async-graphql.

Reply:
Maybe this link would help us in the future? https://github.com/async-graphql/async-graphql/issues/1474